Microsoft.HealthVault.Fhir/Codes/HealthVault/HealthVaultVitalStatisticsCodes.cs

```diff
@@ -15,14 +16,14 @@ namespace Microsoft.HealthVault.Fhir.Codes.HealthVault
     /// </summary>
     public static class HealthVaultVitalStatisticsCodes
```

Comment:
>HealthVaultVitalStatisticsCodes [](start = 24, length = 31)

do you need this class .. this is going round about ? .. this class references HealthVaultVocabularies class and HealthVaultVocabularies in turn references this class ,, can we make it simple by removing this class and put it in HealthVaultVocabulaires itself ?